tests/falco/commands_test.go

```diff
@@ -53,15 +53,27 @@ func TestFalco_Cmd_Version(t *testing.T) {
 		res := falco.Test(runner, falco.WithArgs("--version"))
 		assert.NoError(t, res.Err(), "%s", res.Stderr())
 		assert.Equal(t, res.ExitCode(), 0)
+		// Falco version supports:
+		// - (dev) -> 0.36.0-198+30aa28f
+		// - (release) -> 0.36.0
+		// - (release-rc) -> 0.36.0-rc1
+		// Libs version supports:
+		// - (commit hash) -> e999e61fa8f57ca8e9590e4c108fd4a12459ec48
+		// - (release) -> 0.13.0
+		// - (release-rc) -> 0.13.0-rc1
+		// Default driver supports:
+		// - (commit hash) -> e999e61fa8f57ca8e9590e4c108fd4a12459ec48
+		// - (release) -> 6.0.1+driver
+		// - (release-rc) -> 6.0.1+driver-rc1
```

Comment:
can this happen? :thinking:

Reply:
I would say yes :thinking:

Reply:
Changed `6.0.1+driver-rc1` into `6.0.1-rc1+driver`, thanks!